examples/react-static/src/pages/404.js

```diff
@@ -1,7 +1,9 @@
 import React from 'react'
 
-export default () => (
+const notFound = () => (
```

Comment:
initial cap ins conventional for component names

```suggestion
const NotFound = () => (
```